src/routes/index.js

```diff
@@ -12,6 +13,13 @@ const Routes = () => {
         <Route path='/' exact component={LoginPage} />
         <SecureRoute path='/dashboard' component={Dashboard} />
         <SecureRoute path='/users' component={ListUser} admin />
+        <SecureRoute path='/companies' exact component={Dashboard} admin />
```

Comment:
Just to test the redirect, it will be replaced for the right route when this [pr](https://github.com/comarev/comarev-dashboard/pull/21) is merged